v3.go

```diff
@@ -216,12 +219,18 @@ func RunSyncV3Server(h http.Handler, bindAddr, destV2Server, tlsCert, tlsKey str
 
 	// Block forever
 	var err error
-	if tlsCert != "" && tlsKey != "" {
-		logger.Info().Msgf("listening TLS on %s", bindAddr)
-		err = http.ListenAndServeTLS(bindAddr, tlsCert, tlsKey, srv)
+	if strings.HasPrefix(bindAddr, "/") {
```

Comment:
Docs need updating to mention that `SYNCV3_BINDADDR` can now be a unix socket. Same with `SYNCV3_SERVER`.